website/frontend/src/components/pages/Message/Message.tsx

```diff
@@ -123,8 +119,10 @@ export const Message: FC = () => {
       </div>
     </div>
   ) : (
-    <div className="message">
-      <Loader color="#fff" className="animation-rotate" />
-    </div>
+    <p>
```

Comment:
this is wrong, [you cannot wrap `div` by `p`.](https://stackoverflow.com/questions/8397852/why-cant-the-p-tag-contain-a-div-tag-inside-it)